admin-dev/themes/new-theme/js/app/widgets/ps-number.vue

```diff
@@ -92,11 +92,11 @@
       },
       increment() {
         const value = Number.isNaN(this.value) ? 0 : Number.parseInt(<string> this.value, 10);
-        this.$emit('change', Number.isNaN(value) ? 1 : value + 1);
+        this.$emit('arrowClick', Number.isNaN(value) ? 1 : value + 1);
```

Comment:
Hmmm since this component is generic you shouldn't change the event here, although it may fix the issue for stock it can break behaviours in other places that use it

Reply:
Why do you need a dedicated even? The change event didn't work?